husarion_ugv_gazebo/launch/simulate_robot.launch.py

```diff
@@ -217,6 +239,11 @@ def generate_launch_description():
             "world",
             "--child-frame-id",
             child_tf,
+            "--ros-args",
```

Comment:
Where also. This is just static publisher. We can skip DEBUG for this

Reply:
Skipped